editoast/src/client/runserver.rs

```diff
@@ -36,13 +36,13 @@ pub struct RunserverArgs {
     address: String,
     #[command(flatten)]
     core: CoreArgs,
-    /// If this option is set, any role and permission check will be bypassed. Even if no user is
+    /// If this option is set to false, any role and permission check will be bypassed. Even if no user is
     /// provided by the request headers of if the provided user doesn't have the required privileges.
     // TODO: once the whole role system will be deployed, the default value of this option should
     // be set to false. It's currently set to true in order to pass integration tests, which otherwise
     // only recieve 401 responses.
```

Comment:
Not part of your PR but since we're here, can you also fix this? Thank you.

```suggestion
    // only receive 401 responses.
```

Reply:
Done.